### ISSUE TITLE
`curl` in verification section of module 8 throws error regarding 'setting certificate verify locations'

### ISSUE THREAD

**Issue by rohanbk** [kelseyhightower/kubernetes-the-hard-way#426]
When I execute the following curl command in the [Verification section of Module 8](https://github.com/kelseyhightower/kubernetes-the-hard-way/blob/master/docs/08-bootstrapping-kubernetes-controllers.md#verification-1) from the home directory of each controller instance, `curl --cacert ca.pem https://${KUBERNETES_PUBLIC_ADDRESS}:6443/version`, I get the following error: 

> curl: (77) error setting certificate verify locations:
>   CAfile: ca.pem
>   CApath: /etc/ssl/certs

However, when I execute the curl command with the path to the ca.pem file, I get the expected outcome. `curl --cacert /etc/etcd/ca.pem https://${KUBERNETES_PUBLIC_ADDRESS}:6443/version`

Question: Should this curl command be amended or did I miss some step regarding the CAPath? 

tl;dr: 
- Works as expected: `curl --cacert /etc/etcd/ca.pem https://${KUBERNETES_PUBLIC_ADDRESS}:6443/version`
- Doesn't work as expected: `curl --cacert ca.pem https://${KUBERNETES_PUBLIC_ADDRESS}:6443/version`

**Comment by cgroeschel** [comment]
Also stumbled over that but then I read the section again and noticed the note earlier:

> The compute instances created in this tutorial will not have permission to complete this section. Run the following commands from the same machine used to create the compute instances.
